src/sqlancer/mysql/gen/datadef/MySQLIndexGenerator.java

```diff
@@ -160,4 +124,46 @@ private void indexType() {
     public void setNewSchema(MySQLSchema schema) {
         this.schema = schema;
     }
+    private String generateIndexColumns(MySQLTable table, MySQLExpressionGenerator gen) {
```

Comment:
For such refactorings, it would be great to add a separate commit or PR. This makes reviewing easier, as it is clearer whether something is a new feature or should preserve the existing functionality.